locales/gaia-toolbar.en-US.properties

```diff
@@ -0,0 +1 @@
+gaiaToolbarMore.ariaLabel = More
```

Comment:
If this is purely for example purposes I suggest that we move the demo and these locale files to `examples/demo`. This way users don't get source code mixed up with example code. What you think?